.env.example

```diff
@@ -24,10 +25,11 @@ FEATURE_TOGGLE_CONFIG='{"url":null,"apiKey":null,"static":{}}'
 BANK_ACCOUNT_REQUEST_EMAIL_TARGET=help@doma.ai
 
 # FIREBASE_CONFIG_JSON contents can be retrieved from https://console.firebase.google.com/project/<PROJECT_ID>/settings/serviceaccounts/adminsdk
+# nosemgrep: generic.secrets.security.detected-google-gcm-service-account.detected-google-gcm-service-account
```

Comment:
same?

Reply:
Semgrep not so smart. It recognise google config but not checking actual values. That means in order to make it "fixed" we have to put "# nosemgrep:" comment